Python/View_and_Combine_STL/view.py

```diff
@@ -0,0 +1,15 @@
+from stl import mesh
+from mpl_toolkits import mplot3d
+from matplotlib import pyplot
+
+def view_stl(file_dir):

```

Comment:
Add a function docstring to document what the function is doing

Reply:
Resolved